sdk/tests/test_finbourne_identity.py

```diff
@@ -1,18 +1,30 @@
 import unittest
-from finbourne_identity import api as la, ApiClientBuilder
-from finbourne_identity.utilities import ApiClientFactory
+from finbourne_identity import ApiClientBuilder as identity_api
+from finbourne_identity import api as iapi
+from lusidjam import RefreshingToken
+import lusid
 
 
 class Lydia(unittest.TestCase):
 
     @classmethod
     def setUpClass(cls):
-        api_client = ApiClientFactory()
-        cls.domains_api = api_client.build(la.DomainsApi)
+        lusid_api_factory= lusid.utilities.ApiClientFactory(token=RefreshingToken(),api_secrets_filename="secrets.json")
```

Comment:
You shouldn't need `lusid`. Are you're just importing to get the access token? The same authentication details are valid for lusid and identity/access